conf/routes

```diff
@@ -0,0 +1,23 @@
+# Routes
+# This file defines all application routes (Higher priority routes first)
+# https://www.playframework.com/documentation/latest/ScalaRouting
+# ~~~~
+
+# An example controller showing a sample home page
+GET     /                           controllers.HomeController.index()
+
+GET     /ticketBooking              controllers.TicketBooking.ticketBookingInit
```

Comment:
what if we'd like to expose the API for external application (like a mobile one)? What do you think you would have to do?

Reply:
Unfortunately, I do not know what would have to be done then.